tests/unit/pages/Home.spec.js

```diff
@@ -369,13 +370,7 @@ describe("Home.vue", () => {
     // wait until the DOM updates after promises resolve
     await flushPromises();
 
-    mockAxios.mockResponse(
-      {
-        access_token: "",
-        expires_in: "",
-      },
-      mockAxios.queue()[0]
-    );
+    mockAxios.mockResponse(dummyPublishedPlio, mockAxios.queue()[0]);
```

Comment:
why was this change made? This is not related to items right?

Reply:
In home page here's what my understanding is for the flow when a plio is deleted:
1. A plio delete action is triggered - it makes a "DELETE" request.
2. A new query is made to get the list of all plios
3. For each plio, `getPlio` is requested

This line `373` is supposed to mock the getPlio response. The previous version of the `getPlio` had two promises inside `Promise.all()`, but because of the mockResponse here (with only one JSON), the promises didn't resolve. The test case finished without error.

In the new `getPlio`, as we only request for plio, we need a dummyPlio as the mock API response.